packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -329,7 +329,7 @@ class FlutterError extends AssertionError {
       '_FakeAsync',
       '_FrameCallbackEntry',
     ];
-    final RegExp stackParser = new RegExp(r'^#[0-9]+ +([^.]+).* \(([^/]*)/[^:]+:[0-9]+(?::[0-9]+)?\)$');
+    final RegExp stackParser = new RegExp(r'^#[0-9]+ +([^.]+).* \(([^/]*)/.+:[0-9]+(?::[0-9]+)?\)$');
```

Comment:
If this function will be called more than once, then I believe there are performance benefits in JIT mode in caching a single instance of this RegExp in a lazily allocated static. It looks like this function might only be called once when an exception is thrown though?

Reply:
The function is called once per exception to clean up stack trace, yes.